Week2/solution/solution 2.txt

```diff
@@ -0,0 +1,35 @@
+First we need to write (pragma foreign_key = on)
+
+create table stud183
+(
+	id number(20) primary key,
+	name varchar(20) not null
+);
+
+create table stud_detail183
+(
+	id number(20) references stud56(id),
+	city varchar(20) not null,
+	mobile number(12) not null
+);
+
+
+insert into stud183
+values(1,"bhavin"),(2,"pal"),(3,"priyank"),(4,"hiten"),(5,"krutarth");
+
+insert into stud_detail183
+values(1,"surat",6527848152),(2,"bardoli",9909393113),(3,"kadodara",6353954397),(4,"bagumra",7265070790),(5,"bardoli",9987585940);
+
+1.Display all record of stud table.
+-> select * from stud183;
+
+2.Display all record of stud_detail table.
+-> select * from stud_detail183;
+
+3.Display record of student whose come from city "surat".
+-> select s.id , s.name from stud183 s , stud_detail183 sd
+   where s.id=sd.id and sd.city like  "surat";
+
+4.Display record of student whose come from city start from b.
+-> select s.id , s.name from stud183 s , stud_detail183 sd

```

Comment:
It seems like that there is an error in this line as you written an 's' without any reason.